common/src/Model/Node.h

```diff
@@ -56,7 +56,20 @@ namespace TrenchBroom {
             virtual ~Node();
         public: // getters
             const String& name() const;
+            /**
+             * Returns a box that encloses the node and its children. These are the bounds that are relevant
+             * for gameplay (for entities, the bounds specified in the entity definition file), and used
+             * for grid snapping, for example.
+             * Nodes can render or hit test outside of these bounds if necessary (see `cullingBounds()`).
+             */
             const vm::bbox3& bounds() const;
+            /**
+             * Returns a box that encloses all rendering and hit testing for this node and its children.
+             * Equal to or larger than `bounds()`.
+             * Currently, the only case where this differs from `bounds()` is with entity models that extend
+             * beyond the bounds specified in the .fgd.
+             */
+            const vm::bbox3& cullingBounds() const;
```

Comment:
I am not a fan of the term `culling` in this context. It may be because I am not a native speaker, but culling invokes the idea of pruning objects from a data structure, which is not what's happening here. These bounds are used to find candidates for picking, or hit testing. I would just rename this to `pickingBounds`, but maybe there's a better term still.

Reply:
You're right, it's not a very good name, I was just trying to indicate that if you needed bounds for frustum culling, or culling rays, these are the bounds you would use. `pickingBounds` would be fine / better?